pkgs/by-name/da/dartsim/package.nix

```diff
@@ -47,46 +52,43 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-ik6FwrN5Ta1LinrXpZZc7AmzdFPoLjG07/zo1IZdmgI=";
   };
 
-  # disable failing tests. CMAKE_CTEST_ARGUMENTS does not work.
-  patches = [ ./disable-failing-tests.patch ];
+  patches = [
+    # disable failing tests. CMAKE_CTEST_ARGUMENTS does not work.
+    ./disable-failing-tests.patch
+    # Fix use of system gbenchmark, merged upstream
+    (fetchpatch {
+      url = "https://github.com/dartsim/dart/pull/1904.patch";
+      hash = "sha256-i8Ga0FGVQ3OMprEoGEwVy0j139wjnmR6ABxr/3syhzw=";
+    })
+    # Fix use of system pybind11, merged upstream
+    (fetchpatch {
+      url = "https://github.com/dartsim/dart/pull/1907.patch";
+      hash = "sha256-T3992uD0Z36tTxlcFaikVaLt08N9EP4gOHP0Y2AFBzQ=";
+    })
+    # fix use of absolute CMake paths in .pc, merged upstream
+    (fetchpatch {
+      url = "https://github.com/dartsim/dart/pull/2006.patch";
+      hash = "sha256-csWh0261nbnLWRPqq6QHlG54ANABFmOOrJqOtpGr5dQ=";
+    })
```

Comment:
For reproducibility please point at a commit url instead of a pr number url, and link to the PR in a comment. While i'm not aware of any way to change the patch post-merge it does not mean it is impossible.